design-tokens/build.js

```diff
@@ -141,8 +141,8 @@ function buildThemedTokens(themedTokenGroups, baseDesignTokenGroup) {
                 fileHeader: 'licence-header',
                 selector:
                   theme.name === DEFAULT_THEME
-                    ? ':root'
-                    : `html[${THEME_DATA_ATTRIBUTE}='${theme.name}'], [${THEME_DATA_ATTRIBUTE}='${theme.name}']`,
+                    ? `:root, [${THEME_DATA_ATTRIBUTE}='${theme.name}']`
+                    : `[${THEME_DATA_ATTRIBUTE}='${theme.name}']`,
```

Comment:
I think I'm wrong. For some reason, this might be required specifically for the `html` tag.

Reply:
hmm... I just tried,

```html
<html data-test="hi">
  <head>
    <style>
      [data-test="hi"] {
        background: red;
      }
    </style>
  </head>
</html>
```

in a Codepen and it worked 🤔

Reply:
Ah, I discovered the issue. `:root` selects the document element, so the order in which `:root` and `[data-echoes-theme='dark']` appear in the CSS file matters. By adding `html`, it increases the specificity so that `html[data-echoes-theme='dark']` always beats `:root`, no matter which order they appear in the CSS file.

Reply:
Ah yeah good catch!
It might be worth adding a comment in there to make sure the next person wont have to go through the same research.